lib/temporal.explorer.ts

```diff
@@ -82,70 +81,59 @@ export class TemporalExplorer
       } as WorkerOptions;
       if (connectionOptions) {
         this.logger.verbose('Connecting to the Temporal server');
-        workerOptions.connection = await NativeConnection.connect(connectionOptions);
+        workerOptions.connection = await NativeConnection.connect(
+          connectionOptions,
+        );
       }
 
       this.logger.verbose('Creating a new Worker');
       this.worker = await Worker.create(
-        Object.assign(
-          workerOptions,
-          workerConfig,
-        ),
+        Object.assign(workerOptions, workerConfig),
       );
     }
   }
 
   getWorkerConfigOptions(name?: string): WorkerOptions {
-    return this.moduleRef.get(TEMPORAL_WORKER_CONFIG || name, {
```

Comment:
Removing `{ strict: false }` ensures the options are read within the module scope rather than global scope. This ensures config options can be different, instead of pulling in the last options to be declared.

Reply:
`moduleRef` is no longer needed since the options are directly injected.